.github/workflows/sc-google-code-style-check.yml

```diff
@@ -10,4 +10,4 @@ jobs:
       - uses: actions/checkout@v2 # v2 minimum required
       - uses: axel-op/googlejavaformat-action@v2.0.0
         with:
-          args: "--replace"
\ No newline at end of file
+          args: "--dry-run --set-exit-if-changed"
```

Comment:
maybe add more info in the commit body? I'm not sure whats going on with this

Reply:
added